.gitignore

```diff
@@ -83,3 +83,5 @@ dist
 .idea
 .DS_Store
 assets/.DS_Store
+
+credentials.yml
```

Comment:
Kannst du eine `credentials.yml.example` in das git committen, die Platzhalter-Werte enthält?

Reply:
gute Idee

Reply:
Deswegen kopiert man die `*.yml.example` nach `*.yml`.
Wenn jemand die echten credentials in die `.example` schreibt, ist das schon grob fahrlässig 🤔

Reply:
meinetwegen, ist dann nicht meine schuld wenn es passiert